<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab7CamViewerThread</class>
 <widget class="QWidget" name="Tab7CamViewerThread">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
     <item>
      <widget class="QLabel" name="plabelCamView">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Images.qrc">:/Images/Images/initDisplay.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,2,4">
       <item>
        <widget class="QPushButton" name="pPBcamStart">
         <property name="text">
          <string>CamStart</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pCbRgb">
         <property name="text">
          <string>RGB Classfy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pPBsnapShot">
         <property name="text">
          <string>Snapshot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
